### PR TITLE
Bump version to 14-4.0 - D2D <3

### PR DESCRIPTION
**Pull request by chirayudesai**

### CODE REVIEW

**Review by stevesoltys — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+## [14-4.0] - 2024-12-17
+* D2D
```
Comment (grote) [inline CHANGELOG.md]: Who goes through the change? :P

Comment (chirayudesai) [reply CHANGELOG.md]: List I went through:

```
888b6af7 Bump version to 14-4.0 - D2D <3
0319d733 Merge pull request #562 from seedvault-app/feature/d2d-transfer
daaf0e0e Add experimental support for forcing D2D transfer backups
57adc57c Bump version and set property for testing
57be15ab colors: Switch to public APIs for colors
c2f73745 Initial support for backup of D2D-only apps
5d191606 Pretend to be a device to device data transport
640f5b73 Fix double start of restore session
d7f985ba Add some comments to storage Restore
18013c03 Support installing via F-Droid Basic
39280927 Merge pull request #563 from seedvault-app/bugfix/binder-exception-too-many-packages
b498dab9 Apply latest code review suggestions
9aa57cac Merge pull request #591 from Uldiniad/android14
97d2d18a fixup! Allow secondary user backup to USB
b1a2c6d4 Apply code review suggestions
6c7afd5f Fix binder exception when restoring a large number of applications
a091142a Allow restoring data for a greater number of apps
6ce2e27a Merge pull request #585 from seedvault-app/feature/kotlin-gradle-build-scripts
b131aaf8 Clean up Gradle build scripts
b917293b Switch to Kotlin build scripts + version catalog
ee3a9230 Merge pull request #582 from seedvault-app/feature/e2e-test-logs
551538fd Fix AOSP build artifact location
8297bb70 Fix and improve E2E tests
28059a50 Merge pull request #581 from grote/14-version-upgrades
09bc8f03 Merge pull request #586 from seedvault-app/feature/aosp-build
1a20f634 Build Seedvault with AOSP in CI
f8b94c5f Upgrade tink to 1.10.0
cb95444a Bump minSdk to 33 and upgrade unit test dependencies
74566b90 Upgrade kotlin-bip39-jvm to 1.0.6
bc90378e Merge pull request #579 from seedvault-app/staging/android14-bp
0cd6d05b Merge pull request #576 from seedvault-app/staging/android14
6cb3b30b restore: Runtime-registered broadcasts receivers must specify export behavior
aa19ccd5 Make e2e tests run on pull requests
aa0e82cf ApkInstaller: Supress NewApi for
1e2d66fc Pin Android r8 version until we're on AGP 8.2
978ab689 14: Bump to JavaVersion.VERSION_17
eaa3715a 14: Match gradle dependencies with AOSP android-14.0.0_r1
a6b32ac3 14: Update AOSP libs from gradle, from android-14.0.0_r1
82712487 gradle: Bump to SDK 34
3938549a Cancel previous GitHub actions runs when PR updated
d678d9a9 Bump emulator image in Gradle task to android-34
bf5dc195 Switch to GitHub actions for E2E tests
fcdacf22 14: Add new permission FOREGROUND_SERVICE_DATA_SYNC
e6f32510 Bump version to 14-3.3
```

Comment (grote) [reply CHANGELOG.md]: what about new color scheme, backup size and stop backing up APKs of excluded apps?

Comment (chirayudesai) [reply CHANGELOG.md]: Done

**Review by grote — APPROVED** [review]

**Pull request merged**